src/plot.jl

```diff
@@ -85,7 +86,14 @@ struct Fan; x; y; tipannotations; marker_x; marker_y; showtips; tipfont; marker_
             end
         end
     end
-    dend.showtips && (annotations := map(x -> (x[1], x[2], (x[3], :left, dend.tipfont...)), dend.tipannotations))
+    if dend.showtips
+        @series begin
+            seriestype := :scatter
+            markersize := 0
+            series_annotations := map(x -> text(x[3], :left, dend.tipfont...), dend.tipannotations)
```

Comment:
do you need the `text` here?

Reply:
for this solution, yes. It writes out the unformatted entire tuple otherwise.